node_modules/oae-core/about/css/about.css

```diff
@@ -0,0 +1,62 @@
+
+#about-description .about-description-empty,
```

Comment:
All rules should be prefixed with `.about-widget` (or `.aboutgroup-widget` once renamed)